images/disks-images-provider/entrypoint.sh

```diff
@@ -27,7 +27,17 @@ mkdir -p /images/datavolume1 /images/datavolume2 /images/datavolume3
 echo "converting cirros image from qcow2 to raw, and copying it to local-storage directory, and creating a loopback device from it"
 # /local-storage will be mapped to the host dir, which will also be used by the local storage provider
 qemu-img convert -f qcow2 -O raw /images/cirros/disk.img /local-storage/cirros.img.raw
+
+# Check if attached loopdevice reach limit number (100)
+num=$(losetup -l|wc -l)
+if [ ${num} -gt 100 ];then
+echo "attached loopdevices have reach limit number(100)"
+exit 1
+fi
```

Comment:
I think this will work as well: `[ ${num} -gt 100 ] && exit 1`

Reply:
I wouldn't even have added this, but, since it's in, I think the value here is the message being echo'ed.

Reply:
> I think this will work as well: `[ ${num} -gt 100 ] && exit 1`

If use this, i can not echo message. Can I keep my version?

Reply:
I'd prefer have your lovely error message to signal me when all hell broke loose. IMO, we should keep your version.

Reply:
You mean: `[ ${num} -gt 100 ] && echo "...."; exit 1` ?

Reply:
> You mean: `[ ${num} -gt 100 ] && echo "...."; exit 1` ?

Got it.

Reply:
> You mean: `[ ${num} -gt 100 ] && echo "...."; exit 1` ?
It will keep failing as `exit 1` always run 
How about this one?
`([ $(losetup -l|wc -l) -gt 100 ] && echo "attached loopdevices have reach limit number(100)") && exit 1`

Reply:
Yea, sorry for the `;`.